contents/docs/posthog-ai/platform-and-chat-ui.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: 'PostHog AI: platform and chat UI'
+---
+
+import { IconSparkles } from '@posthog/icons'
+
+PostHog AI powers all AI features within the [PostHog app](https://app.posthog.com). It is an agentic system and intelligence layer that sits on top of the PostHog platform. As a result, there are multiple ways of working with PostHog AI.
+
+Here's an overview of the different UIs for interacting with PostHog AI.
+
+## In-app chat
+
+The primary dedicated interface is an in-app <a href="https://app.posthog.com/#panel=max">AI chat</a> for conventional prompting and conversation-based flows.
+
+When logged in, the PostHog AI chat is always available in the right-hand sidebar of the PostHog app. You can open it at any time by clicking the PostHog AI icon.
+
+<ProductVideo
+    videoLight="https://res.cloudinary.com/dmukukwp6/video/upload/posthog_ai_sidebar_4k_compressed_75af4f4370.mp4" 
+    alt="PostHog AI chat" 
+    classes="rounded"
+    autoPlay={true}
+/>
+
+<Caption>
+The PostHog AI chat is always available in the sidebar
+</Caption>
+
+The in-app chat supports conversation history and features like [context, commands, and memory](/docs/posthog-ai/context-and-commands).
+
+## Platform UI
+
+PostHog AI shows up throughout the PostHog platform as inline or ambient UI elements, typically labelled with the <span className="inline-block"><IconSparkles className="h-4 w-4" /> </span> icon, and embedded within other features.
```

Comment:
Proposing we frame these elements as "entrypoints", i.e. they always are an initial prompt for the AI chat